docs/migration-guide-2.9-2.10.md

```diff
@@ -2,5 +2,5 @@
 title: Migration Guide 2.9-2.10
 ---
 
-Due to the removal of the `kyma-integration` Namespace, we need to migrate all Application Connector Secrets to the `kyma-system` Namespace. After upgrading the Kyma to the 2.10 version, you need to execute the following script [2.9-2.10-OS-copy-secrets-to-system-namespace.sh](./assets/2.9-2.10-OS-copy-secrets-to-system-namespace.sh).
+Due to the removal of the `kyma-integration` Namespace, we need to migrate all Application Connector Secrets to the `kyma-system` Namespace. After upgrading the Kyma to the 2.10 version, you need to execute the following script [2.9-2.10-OS-copy-secrets-to-system-namespace.sh](https://github.com/kyma-project/kyma/blob/main/docs/assets/2.9-2.10-OS-copy-secrets-to-system-namespace.sh).
```

Comment:
```suggestion
Due to the removal of the `kyma-integration` Namespace, we need to migrate all Application Connector Secrets to the `kyma-system` Namespace. After upgrading Kyma to version 2.10, you need to execute the following script [2.9-2.10-OS-copy-secrets-to-system-namespace.sh](https://github.com/kyma-project/kyma/blob/main/docs/assets/2.9-2.10-OS-copy-secrets-to-system-namespace.sh).
```

Reply:
I'm not sure about `upgrading THE Kyma`. I don't think we need an article there.
I'm all aboard with the second one :)

Reply:
> I'm not sure about `upgrading THE Kyma`. I don't think we need an article there. I'm all aboard with the second one :)
Well, I've expressed my opinion in the suggestion 😄